eng/README.md

```diff
@@ -49,9 +49,21 @@ release branch is created, someone with `darc` installed locally will need to
 run the `add-subscription` command to configure updates against that new branch.
 
 
+#### Build Asset Manifest Promotion
+
+Builds from main and release branches will push NuGet package metadata to the
+darc/maestro Build Asset Registry.  This build information will also be promoted
+to a default darc/maestro channel if one is configured.  Default channels are
+manually managed at this time.  To configure a new default repo+branch <-> channel
+association, run the [`darc add-default-channel`][6] command:
+```
+darc add-default-channel --channel ".NET 6" --branch "main" --repo https://github.com/xamarin/xamarin-android
```

Comment:
Will this impact branching at all? Will we have to do anything if we make a .NET 6 Preview 4 branch?

Reply:
This may be worth mentioning in the branch notes document, if we want the darc/maestro dependency flow to work when we branch we will need to configure new default channels and subscriptions:

```
darc add-default-channel --channel ".NET 6.0.1xx SDK Preview 4" --branch "6.0.1xx-preview4" --repo https://github.com/xamarin/xamarin-android

darc add-subscription --channel ".NET 6.0.1xx SDK Preview 4" --source-repo https://github.com/xamarin/xamarin-android --target-repo https://github.com/dotnet/maui --target-branch 6.0.1xx-preview4 --update-frequency everyDay
```